src/test/java/core/basesyntax/strategy/OperationStrategyTest.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.strategy;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.operation.BalanceOperationHandler;
+import core.basesyntax.strategy.operation.BuyOperationHandler;
+import core.basesyntax.strategy.operation.OperationHandler;
+import core.basesyntax.strategy.operation.ReturnOperationHandler;
+import core.basesyntax.strategy.operation.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class OperationStrategyTest {
+    private static Map<FruitTransaction.Operation, OperationHandler> operationHandlerMap;
+
+    @BeforeClass
+    public static void beforeClass() {
+        operationHandlerMap = new HashMap<>();
+        operationHandlerMap.put(FruitTransaction.Operation.BALANCE, new BalanceOperationHandler());
+        operationHandlerMap.put(FruitTransaction.Operation.PURCHASE, new BuyOperationHandler());
+        operationHandlerMap.put(FruitTransaction.Operation.RETURN, new ReturnOperationHandler());
+        operationHandlerMap.put(FruitTransaction.Operation.SUPPLY, new SupplyOperationHandler());
+    }
+
+    @Test
+    public void operationStrategy_getOperation_Ok() {
+        OperationHandler expected = operationHandlerMap.get(FruitTransaction.Operation.BALANCE);
+        OperationStrategy operationStrategy = new OperationStrategyImpl(operationHandlerMap);
```

Comment:
u should build your test case over the OperationStrategy, not only over operationHandlerMap